.github/workflows/test/linux-nightly-ci.yml

```
@@ -0,0 +1,101 @@
+            xfonts-base \
+            nickle cairo-5c \
+            xorg-docs-core
+      - name: Set DISPLAY environment variable
+        run: |
+          Xvfb :99 -screen 0 1920x1080x24 &
+          echo "DISPLAY=:99" >> $GITHUB_ENV
+      - name: Start D-Bus
+        run: |
+          dbus-launch --exit-with-session &
+      - name: Start Dbus
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values

(shellcheck)